app/src/main/java/com/infomaniak/mail/views/itemViews/AvatarMergedContactData.kt

```diff
@@ -43,4 +48,13 @@ class AvatarMergedContactData @Inject constructor(
         .getMergedContactsAsync()
         .mapLatest { ContactUtils.arrangeMergedContacts(it.list.copyFromRealm()) }
         .asLiveData(ioCoroutineContext)
+
+    val isBimiEnabledLiveData = mailboxController
+        .getMailboxAsync(AccountUtils.currentUserId, AccountUtils.currentMailboxId)
+        .mapLatest {
+            val mailbox = it.obj
+            mailbox?.featureFlags?.contains(FeatureFlag.BIMI)
+        }
+        .filterNotNull()
+        .asLiveData(ioCoroutineContext)
```

Comment:
Add a `distinctUntilChanged()` so the live data will ignore all the small mailbox changes that don't impact feature flags